dask_kubernetes/operator/controller/controller.py

```diff
@@ -462,6 +462,7 @@ async def get_desired_workers(scheduler_service_name, namespace, logger):
 
 
 @kopf.on.field("daskworkergroup.kubernetes.dask.org", field="spec.worker.replicas")
+@kopf.on.delete(labels={"dask.org/component": "worker"})
```

Comment:
```suggestion
@kopf.on.delete("pod", labels={"dask.org/component": "worker"})
```

Reply:
Looks like you missed the resource type. It also looks like you haven't allowed maintainers to push to your branch, so I can't quickly commit this. Could you update this and push again?